docs/concepts/dependencies.md

```diff
@@ -70,6 +70,7 @@ stands-compliant `project.dependencies` table.
 During development, a project may rely on a package that isn't available on PyPI. The following
 additional sources are supported by uv:
 
+- Index: A package from an explicit package index.
```

Comment:
I worry the use of "explicit" here will be conflated with the `explicit` index marker